keystone/files/mitaka/keystone.conf.Debian

```diff
@@ -2193,3 +2193,11 @@ driver = memcache_pool
 
 [extra_headers]
 Distribution = Ubuntu
+
+{% for section in server.get('custom_sections', {}) %}
+[{{ section }}]
```

Comment:
This is definately not what we want to do - allow user to define anything without any validation etc.
Create specific metadata for what you are trying to achieve.